### DISCUSSION TITLE
Moving rows between tables [Jun 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6024] [Discord Threads]
> 🙋‍♂️ **rimehaug**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Can you elaborate on your use case?    
#### You might want to organise information properly in your table to reduce duplication & create link between records.    
####    
#### https://docs.nocodb.com/setup-and-usages/link-to-another-record  
#  
> 🙋‍♂️ **rimehaug**  
#### So, I have 1 excel where I create accounts, and create activity on them.    
#### Then another excel where I use those accs for various reasons for clients.    
####    
#### So then I just copy the fields over to the other excel, and mark it used.    
####    
#### I believe I have the wrong idea for how this should be done in NocoDB and I'm trying to replicate something that should be done differently  
#  
> 🙋‍♂️ **rimehaug**  
#### This is one of the excels  
![image0](https://cdn.discordapp.com/attachments/1111300348263530536/1111563098004918323/image.png)  
#  
> 🙋‍♂️ **rimehaug**  
#### Then copied to this excel to use:  
![image0](https://cdn.discordapp.com/attachments/1111300348263530536/1111563370638876752/image.png)  
#  
> 👨‍🦲 **dstala**  
#### Good to use Links instead of duplicating records when working with database. It simplifies managing larger data set since there is no redundancy.    
####    
#### From an outlook, it looked to me that you can manage all details in one sheet itself  
#  
> 🙋‍♂️ **rimehaug**  
#### Yes this was my thought too. Thanks for the response!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1111300348263530536)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
